build/setEnv.sh

```diff
@@ -7,31 +7,5 @@ if [ ! -f "build/setEnv.sh" ]; then
     exit 2
 fi
 
-    echo "Creating a fake go workspace, so that user can build from source placed in non-go directory" 
-    tempGoPath="$PWD/build/workspace_"
-
-    #Any change in project path should be reflected in DST_GO_PATH and tempDir
-    DST_GO_PATH="/src/github.com/direct-state-transfer/dst-go/"
-    export DST_GO_PATH
-    root="$PWD"
-    tempDir="$tempGoPath/src/github.com/direct-state-transfer"
-
-    if [ ! -L "$tempDir/dst-go" ]; then
-     mkdir -p "$tempDir"
-     mkdir -p "$tempGoPath/bin"
-     cd "$tempDir"
-     ln -s ../../../../../. dst-go
-     cd "$root"
-    fi
-   
-    # Set up the environment to use the workspace.
-    export GOPATH="$tempGoPath"
-    export GOBIN=""
-
-    # Change Dir
-    cd "$tempDir/dst-go"
-    PWD="$tempDir/dst-go"   
-    echo "Temp GOPATH set to fake go workspace : $PWD \n"
-
```

Comment:
Does it make sense to get rid of this file, as there is no functionality included anymore?

Reply:
build/setEnv.sh removed in [this commit](https://github.com/direct-state-transfer/dst-go/pull/20/commits/b0fd76b77821d03ba91dc943ba5172d061a2ece1).

Reply:
If the remaining check in build/setEnv.sh, i.e. that build is run from the root directory, is important, you could still put that into the build module...

Reply:
the build package is intended to be used only by Makefile and makefile can be invoked only from project root.
Still, I have added checks in build package in [this commit](https://github.com/direct-state-transfer/dst-go/pull/20/commits/147a8a702fe50e806e586243490de1d377533ca2) to inform the user and exit with error message if invoked from any other directory.